packages/manager/src/features/Databases/DatabaseDetail/DatabaseBackups/DatabaseBackups.tsx

```diff
@@ -94,14 +120,142 @@ export const DatabaseBackups = (props: Props) => {
+
+    return dateStart < backupStart || dateStart > today;
+  };
+
+  const onRestoreNewDatabase = (selectedDate: DateTime | null) => {
+    const day = selectedDate?.toISODate();
+    const time = selectedTime?.toISOTime({ includeOffset: false });
+    const selectedDateTime = `${day}T${time}Z`;
+
+    const selectedTimestamp = new Date(selectedDateTime).toISOString();
+
+    setSelectedRestoreTime(selectedTimestamp);
+    setIsRestoreDialogOpen(true);
+  };
+
+  return isNewDatabase ? (
+    <Paper style={{ marginTop: 16 }}>
+      <Typography variant="h2">Summary</Typography>
+      <StyledTypography>
+        Databases are automatically backed-up with full daily backups for the
+        past 10 days, and binary logs recorded continuously. Full backups are
+        version-specific binary backups, which when combined with binary
+        logs allow for consistent recovery to a specific point in time (PITR).
+      </StyledTypography>
+      {/* TODO: Uncomment when the all data is available (Number of Full Backups, Newest Full Backup, Oldest Full Backup) */}
+      {/* <Hidden xlDown={true} xlUp={true}>
+        <Grid alignItems="stretch" container mt={2} spacing={1}>
+          <Grid item md={4} xs={12}>
+            <StyledBox>
+              <Typography variant="h2">Number of Full Backups</Typography>
+              <Typography component="span" variant="h1">
+                {backups?.data.length}
+              </Typography>
+            </StyledBox>
+          </Grid>
+          <Grid item md={4} xs={12}>
+            <StyledBox>
+              <Typography variant="h2">Newest Full Backup</Typography>
+              <Typography variant="subtitle2">{newestBackup} (UTC)</Typography>
+            </StyledBox>
+          </Grid>
+          <Grid item md={4} xs={12}>
+            <StyledBox>
+              <Typography variant="h2">Oldest Full Backup</Typography>
+              <Typography variant="subtitle2">{oldestBackup} (UTC)</Typography>
+            </StyledBox>
+          </Grid>
+        </Grid>
+      </Hidden> */}
+      <Divider spacingBottom={25} spacingTop={25} />
+      <Typography variant="h2">Restore a Backup</Typography>
+      <StyledTypography>
+        Select a date and time within the last 10 days you want to create a fork
+        from.
+      </StyledTypography>
+      <Grid container justifyContent="flex-start" mt={2}>
+        <Grid item lg={3} md={4} xs={12}>
+          <Typography variant="h3">Date</Typography>
+          <LocalizationProvider dateAdapter={AdapterLuxon}>
+            <StyledDateCalendar
+              shouldDisableDate={(date) =>
+                isWithinBackupTimeframe(date, oldestBackup)
+              }
+              onChange={(newDate) => setSelectedDate(newDate)}
+              value={selectedDate}
+            />
+          </LocalizationProvider>
+        </Grid>
+        <Grid item lg={3} md={4} xs={12}>
+          <Typography variant="h3">Time(UTC)</Typography>
```

Comment:
```suggestion
          <Typography variant="h3">Time (UTC)</Typography>
```
Let's include a space here. Test will need to be updated as well.